### ISSUE TITLE
Custom "schema name" and generate integers as Avro int and not long

### ISSUE THREAD

**Issue by yoavp10** [godatadriven/pydantic-avro#55]
Hi
I want to migrate some manual json schemas into using this great lib :)
For this I need to implement 2 small features:
1. Be able to provide "name" (in the root of the schema)
2. generate python "int" as avro int and not long.

I will happily open the relevant PR if you please provide me permissions.
Thanks

**Comment by ffinfo** [comment]
@yoavp10 In GitHub you can always make PR's if from your own fork of this repo (at least if the original repo is public)

**Comment by ffinfo** [comment]
The main problem with python ints is that they are variable in size, see also this:
```
>>> 2 ** 64
18446744073709551616
>>> type(2 ** 64)
<class 'int'>
```

This is also the reason why it should go be default to long instead of an int. I will see how I can annotate this on a single value. A generic config is not enough because some models maybe will require to use int and longs together in 1 model.

**Comment by ffinfo** [comment]
for the int support I came up with this:
https://github.com/godatadriven/pydantic-avro/pull/59

**Comment by ffinfo** [comment]
If I'm correct all the issue are resolved in version 0.5.1
If not please reopen the issue.

**Comment by yoavp10** [comment]
awesome, thank you very much. will let you know if I see any more issues!
